code/mob/living/carbon/human.dm

```diff
@@ -99,6 +99,7 @@
 
 	var/datum/hud/human/hud
 	var/mini_health_hud = 0
+	var/static/datum/zone_sel_offsets/static_zone_sel_offsets = new(list(0,14),list(0,4),list(-6,0),list(6,0),list(-6,-10),list(6,-10))
```

Comment:
Instead of defining a new static var for each type, use `TYPEINFO`- basically statics (data which is shared on all objects of the type) but you can override them.
See below, bug me or pali if something doesn't make sense (it is a bit of a weird system)
https://github.com/goonstation/goonstation/blob/f081fddc3ad090ae526e67d7ee716a2ea8bbd3a9/_std/types.dm#L251-L273